### ISSUE TITLE
[Feature Request]: Mouse cursor bug prevention

### ISSUE THREAD

**Issue by yeweivon** [ShootMe/FallGuysStats#224]
### What is your request?

I wonder if this feature has been removed permanently or it will come back sometime after? I don't see it in the settings right now. It was quite helpful for mouse and keyboard players if the feature can be turned on.  Thank you!!

**Comment by qutrits** [comment]
I thought the mouse bug went away this season, but is it still there?

**Comment by yeweivon** [comment]
It is still happening but not so often.

Another issue I am having right now is if I have 'always on top' turned on, I can't click on anything when I pull out 'show hidden icons' in the task bar (as I get used to put the overlay at the right bottom corner). Is there a solution to that?

**Comment by qutrits** [comment]
I need a more precise description. If possible, a screenshot or video would be great.

**Comment by yeweivon** [comment]
Here's the video showing the hidden icon blocked by the overlay. There was no issues locating at the corner before. https://github.com/ShootMe/FallGuysStats/assets/132220283/4c009e6e-e6c2-4735-9836-15faaf2e6a9e
